qutebrowser/config/configdata.yml

```diff
@@ -2512,6 +2512,22 @@ colors.webpage.prefers_color_scheme_dark:
 
 # emacs: '
 
+## unkown_url
+
+unknown_url.scheme.policy:
+  default: 1
```

Comment:
The default from Qt is `AllowUnknownUrlSchemesFromUserInteraction`. We should keep that.